packages/block-library/src/post-content/edit.js

```diff
@@ -32,9 +31,9 @@ function ReadOnlyContent( { userCanEdit, postType, postId } ) {
 			<Warning>{ __( 'This content is password protected.' ) }</Warning>
 		</div>
 	) : (
-		<div { ...blockProps }>
-			<RawHTML key="html">{ content?.rendered }</RawHTML>
-		</div>
+		<div { ...blockProps }
```

Comment:
The linting step (Prettier) is complaining that this isn't on a new line, and it also wants an extra space surrounding the object passed to `dangerouslySetInnerHTML` 🙂